frontend/src/api/api.ts

```diff
@@ -135,6 +135,7 @@ export async function chatApiGpt(options: ChatRequestGpt, user: any): Promise<As
             conversation_id: options.conversation_id,
             query: options.query,
             url: options.file_blob_url,
+            agent: options.agent,
```

Comment:
@ramirezmorac2, the agent parameter needs to be sent here because we've already set up the endpoint URL based on the agent. What is missing now is the document name. "documentName"